modules/nw-multi-network-policy-ipv6-suppport.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-multi-network-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-multi-network-policy-ipv6-support_{context}"]
+= Supporting multi-network policies in IPv6 networks
+
+The ICMPv6 Neighbor Discovery Protocol (NDP) plays a crucial role in IPv6 networks, facilitating the interaction between devices on the same link. NDP is a set of messages and processes that enable devices to discover and maintain information about neighboring nodes.
+
+The Cluster Network Operator (CNO) deploys the iptables implementation of multi-network policy when `useMultiNetworkPolicy` is set to `true`.
+
+To support multi-network policies in IPv6 networks the Cluster Network Operator deploys the following set of rules in every pod affected by a multi-network policy.
+
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: multi-networkpolicy-custom-rules
+  namespace: openshift-multus
+data:
+
+  custom-v6-rules.txt: |
+    # accept NDP
+    -p icmpv6 --icmpv6-type neighbor-solicitation -j ACCEPT <1>
+    -p icmpv6 --icmpv6-type neighbor-advertisement -j ACCEPT <2>
+    # accept RA/RS
+    -p icmpv6 --icmpv6-type router-solicitation -j ACCEPT <3>
+    -p icmpv6 --icmpv6-type router-advertisement -j ACCEPT <4>
+----
+
+<1> This rule allows incoming ICMPv6 Neighbor Solicitation messages, which are part of the neighbor discovery protocol (NDP). These messages help determine the link-layer addresses of neighboring nodes.
```

Comment:
In the first paragraph, you use title case (the first letter of each word in Neighbor Discovery Protocol is capitalized). Here, you do not (they're all lower case). Not sure which it should be, but we should be consistent.

Reply:
going with lower case to be consistent